src/svg-icons/add.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable */
+
+import React from 'react';
+import pure from 'recompose/pure';
+import SvgIcon from '../SvgIcon';
+
+let Add = (props) => (
```

Comment:
*Side comment as I don't think it makes sense here:*
We are not longer using arrow function for the components but named function.
```jsx
function Add(props)
```
@nathanmarks I'm curious to know the motivation. From my end,
1. It makes migration to class component easier
2. We are not messing around with the context.